### PR TITLE
Updated API and Forge version

### PR DESCRIPTION
**Pull request by FreeMemeEU**

### CODE REVIEW

File: src/main/resources/plugin_info.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "ChatPlus",
-  "version": "1.1",
+  "version": "2.0",
```
Comment (huddy987) [inline src/main/resources/plugin_info.json]: Lets make the version 1.2 not 2.0, its not that big of a change to require a major version change

**Review by huddy987 — CHANGES_REQUESTED** [review]

File: src/main/resources/plugin_info.json (hunk 1 above)
Comment (FreeMemeEU) [reply src/main/resources/plugin_info.json]: Done

File: gradle.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
 kotlin.code.style=official
 org.gradle.jvmargs=-Xmx3G
 modGroup=com.lambda
-modVersion=1.1
+modVersion=2.0
```
Comment (huddy987) [inline gradle.properties]: Version still isn't updated to 1.2 here

**Review by huddy987 — CHANGES_REQUESTED** [review]

**Comment by huddy987** [comment]
Looks good, apparently I don't have permission to merge though

**Pull request merged**